Makefile

```diff
@@ -31,7 +31,7 @@ endif
 
 PACKAGE_NAME=Linux-Entra-SSO
 
-RELEASE_TAG=$(shell git describe --match "v[0-9].[0-9]*" --dirty)
+RELEASE_TAG ?= $(shell git describe --match "v[0-9].[0-9]*" --dirty)
```

Comment:
Not needed, you can always call `make RELEASE_TAG=...`, and that will be respected. Or what is your use case?

Reply:
Right. Actually I forgot about this option. Nonetheless I consider this `?=` as an interface specification to tell users that they deliberately can change this option. This separates it from other variables assigned with `=`, which are not intended to be changed form the outside. But yes, it's a matter of taste.